config/default.cfg

```diff
@@ -25,7 +25,7 @@ cfg$description <- "REMIND run with default settings"
 cfg$regionmapping <- "config/regionmappingH12.csv"
 
 #### Current input data revision (<mainrevision>.<subrevision>) ####
-cfg$inputRevision <- 6.296006
+cfg$inputRevision <- rev6.298
```

Comment:
seeing this just by chance, isn't the `rev` superfluous here?

Reply:
Course not.  Thanks/